pkg/updates/source.go

```diff
@@ -0,0 +1,27 @@
+package updates
+
+const Head = ""
```

Comment:
why empty string?

Reply:
mostly to be backwards compatible, where the migration being empty was inferred as `head`

Reply:
actually I think I changed everything to explicitly say `head` and this isn't used